app/Http/Controllers/Api/SignupsController.php

```diff
@@ -38,6 +35,22 @@ public function __construct(SignupService $signups, PostService $posts)
     {
         $this->signups = $signups;
         $this->posts = $posts;
+
+        $this->transformer = new SignupTransformer;
+    }
+
+    /**
+     * Returns signups.
+     * GET /signups
+     *
+     * @param Request $request
+     * @return ]Illuminate\Http\Response
+     */
+    public function index(Request $request)
```

Comment:
I think it might be a good idea to move this into a new controller (maybe namespaced in `\Rogue\Http\Controllers\Three`?) since it's a "v3" route.